appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.3.{build}-{branch}
```

Comment:
How about ```1.3.x-proofbuild-{build}```? That version is only displayed on appveyor and branch shows "master" for PRs against master so it'll be pretty useless information in any situation I can think about.